ruby/hashes.rb

```diff
@@ -0,0 +1,95 @@
+design_client = {}
+
+puts "Please enter the following information:"
+
+print "Name: "
+design_client[:name] = gets.chomp.capitalize
+
+
+print "Address: "
+design_client[:address] = gets.chomp.capitalize
+
+
+print "Email: "
+design_client[:email] = gets.chomp.capitalize
+
+
+print "Allegies to materials? (y/n): "
+design_client[:allergies] = gets.chomp.capitalize
+	if design_client[:allergies] == "y"
+		design_client[:allergies] = true
+	else
+		design_client[:allergies] = false
+	end
+
+wallpaper_types = [:paisley, :chevrons, :photo_realistic, :abstract]
```

Comment:
As im looking at this im not sure that the wall paper types should be symbols.  Am I missing somewhere in the assignment where they are supposed to be symbols?

I know the keys are symbols but I do not feel that the values should be symbols in this exercise